src/components/tooltip/tooltip-test.stories.tsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable react/prop-types */
+import React, { forwardRef, useState } from "react";
+import Tooltip from ".";
+import { TOOLTIP_POSITIONS, TooltipPositions } from "./tooltip.config";
+import { ButtonProps } from "../button";
+
+export default {
+  title: "Tooltip/Test",
+  includeStories: ["DefaultStory"],
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
+    type: {
+      options: ["error", "default", "info", "warning"],
+      control: {
+        type: "select",
+      },
+    },
+    size: {
+      options: ["medium", "large"],
+      control: {
+        type: "select",
+      },
+    },
+    flipOverrides: {
+      options: ["left", "right", "top", "bottom"],
+      control: {
+        type: "select",
+      },
+    },
```

Comment:
Currently, if we were to pass `flipOverrides` directly to `Tooltip` within `DefaultStory`, we would get a type error, since `flipOverrides` passed into the story has the type `TooltipPositions` and not `TooltipPositions[]`.

To fix this, we need to map whatever value we select via the storybook controls into an array like so:

```suggestion
flipOverrides: {
      mapping: (value: TooltipPositions) => [value],
      options: ["left", "right", "top", "bottom"],
      control: {
        type: "select",
      },
    },
```

Which means the type of `flipOverrides` will correctly be `TooltipPositions[]` and not `TooltipPositions`.